src/views/Correction/CorpCorrection.vue

```diff
@@ -188,7 +188,7 @@ export default class CorpCorrection extends Mixins(CommonMixin, FeeMixin, Filing
       orgPersons: items[3],
       shareStructure: items[4],
       nameTranslations: items[5],
-      nameResolutions: items[6]
+      resolutions: items[6]
```

Comment:
This gets stored in "entity snapshot" but the store getter is looking for this property name not the old one (which isn't used anywhere in this project). Must have been a typo.